### PR TITLE
Add events

### PR DESCRIPTION
**Pull request by brianmcmichael**
Adds events to the `master` branch as a safety in case the non `0.5.12` contracts get used by an external team.

### CODE REVIEW

File: src/join-7.sol [hunk 1]
```diff
@@ -88,6 +103,7 @@ contract GemJoin7 {
         uint256 wad = mul(sub(gem.balanceOf(address(this)), bal), 10 ** (18 - dec));
         require(int256(wad) >= 0, "GemJoin7/overflow");
         vat.slip(ilk, urn, int256(wad));
+        emit Join(urn, wad);
```
Comment (talbaneth) [inline src/join-7.sol:106]: We probably want to log amt and not wad here. For example in GemJoin8 we log the input parameter and not the decimalss scaled quantity.  Also in exit() in this GemJoin.

Comment (brianmcmichael) [reply src/join-7.sol:106]: Yeah good point. I was going by the slip amounts elsewhere but I agree collateral units are better.

**Review by talbaneth — APPROVED** [review]
LGTM

File: src/join-5.sol [hunk 2]
```diff
@@ -55,7 +55,7 @@ contract GemJoin5 {
     event Rely(address indexed usr);
     event Deny(address indexed usr);
     event Join(address indexed urn, uint256 wad);
```
Comment (talbaneth) [inline src/join-5.sol:57]: here as well :)

**Review by talbaneth — APPROVED** [review]
LGTM

**Pull request merged**